packages/aztec.js/src/signer/index.js

```diff
@@ -44,22 +40,21 @@ sign.generateZKAssetDomainParams = function generateZKAssetDomainParams(verifyin
 
 /**
  * Create an EIP712 ECDSA signature over structured data
- * @method signStructuredData
+ * @method signTypedData
```

Comment:
Why the change?

Reply:
That's how it's defined in the official documentation.